assets/src/edit-story/output/utils/styles.js

```diff
@@ -27,6 +27,14 @@ function isHexColorString(s) {
 function CustomStyles() {
   const safeToFullRatio = PAGE_RATIO / FULLBLEED_RATIO;
   const fullToSafeRatio = 1 / safeToFullRatio;
+  const safeRatio = PAGE_RATIO;
+  const fullRatio = FULLBLEED_RATIO;
+
+  // The grid layer should stick to the top when the device ratio is between
+  // 493 / 1000 and 9 / 16. The ratio 493 / 1000 can guarantee the grid layer
+  // will respond and not having overlaps with the share button.
```

Comment:
This comment is better! Let's make it even better:

"Make the story content aligned with top of viewport between aspect ratios 493:1000 and 9:16. This ensures that the story's system UI (progress bar, share button) is either completely overlapping or not overlapping the story content (i.e. never partially overlapping)."

Reply:
Updated.

Reply:
Thanks for the example!